app/src/main/java/org/amfoss/templeapp/activities/InsertData.java

```diff
@@ -90,14 +116,24 @@ public void onClick(View v) {
                     @Override
                     public void onClick(View view) {
                         id = id + uid1ET.getText().toString();
+                        String sp = String.valueOf(poojaType.getSelectedItem());
                         name = nameET.getText().toString();
                         if (id.length() == 3) {
                             uid1ET.setError(getString(R.string.valid_ID));
                         } else {
                             if (flag == 1) {
-                                poojaTyp = String.valueOf(poojaType.getSelectedItem());
+                                if (sp.contentEquals("Custom Pooja")) {
```

Comment:
can you check by using string array instead of "Custom Pooja" because it is hardcoded.